pkg/cnab/provider/credentials.go

```diff
@@ -6,7 +6,11 @@ import (
 
 	"github.com/deislabs/cnab-go/bundle"
 	"github.com/deislabs/cnab-go/credentials"
-	"github.com/deislabs/duffle/pkg/duffle/home"
+)
+
+const (
+	// CredentialsDirectory represents the name of the directory where credentials are stored
+	CredentialsDirectory = "credentials"
```

Comment:
Same comment as above, seems like this should be a function on Config?

Reply:
Good call; added in most recent commit (function already existed 😅 )